src/util/psbt/macros.rs

```diff
@@ -155,3 +179,33 @@ macro_rules! impl_psbt_get_pair {
         }
     };
 }
+
+// macros for serde of hashes
+macro_rules! impl_psbt_hash_de_serialize {
+    ($thing:ty) => {
+        impl_psbt_hash_serialize!($thing);
+        impl_psbt_hash_deserialize!($thing);
+    };
+}
+
+macro_rules! impl_psbt_hash_deserialize {
+    ($thing:ty) => {
```

Comment:
Instead of `thing` it could probably be called `hash` or `hash_type`? (although the context makes it clear)